examples/cpp/README.md

```diff
@@ -2,3 +2,9 @@ C++ Examples
 ============
 
 These examples demonstrate building C++ binaries, libraries, and tests.
+
+To starting build , change to this directory.
+```
+cd examples/cpp/
+bazel build //:hello-world
```

Comment:
Have you tested this? The `:hello-world` target is in the `//examples/cpp` package. There is no `:hello-world` target in the root BUILD file.